tests/e2e/Services/Database/DatabaseCustomServerTest.php

```diff
@@ -792,7 +792,7 @@ public function testIndexLimitException()
         // testing for indexLimit = 64
         // MariaDB, MySQL, and MongoDB create 3 indexes per new collection
         // Add up to the limit, then check if the next index throws IndexLimitException
-        for ($i = 0; $i < 61; $i++) {
+        for ($i = 0; $i < 59; $i++) {
```

Comment:
Lets update the comment above regarding the created indexes.

Reply:
Sure!